src/sentry/search/events/base.py

```diff
@@ -1,12 +1,12 @@
-from typing import List, Mapping, Set
+from typing import List, Mapping, Optional, Set
 
 from django.utils.functional import cached_property
 from snuba_sdk.column import Column
-from snuba_sdk.function import CurriedFunction
+from snuba_sdk.function import CurriedFunction, Function
 from snuba_sdk.orderby import OrderBy
 
 from sentry.models import Project
-from sentry.search.events.constants import SNQL_FIELD_ALLOWLIST
+from sentry.search.events.constants import SNQL_FIELD_ALLOWLIST, TAG_KEY_RE
```

Comment:
We can do in another PR if you want, but I think with tag support its time we just drop the allowlist altogether now, the only things that aren't "allowed" now are unimplemented aliases&functions I think

Reply:
Removed